test/spec/modules/sonobiBidAdapter_spec.js

```diff
@@ -318,6 +318,10 @@ describe('SonobiBidAdapter', () => {
 
     it('should map bidResponse to prebidResponse', () => {
       const response = spec.interpretResponse(bidResponse, bidRequests);
+      response.forEach(resp => {
+        let regx = /http:\/\/localhost:9876\/.*?(?="|$)/
```

Comment:
I'm not sure hardcoded the port is a good way to prevent this error. I'm not sure but some process is generating the port (karma?) and we should probably use whatever it is assigning or figure out another way to prevent this test case failure.

Reply:
Like we discussed offline - I remove the port values from the unit tests.  Please take another look when possible.